api-tests/apply_route_test.go

```diff
@@ -112,7 +112,7 @@ func TestApplyPsmdbShouldReturnSameMajorVersion(t *testing.T) {
 
 	psmdbParams := &version_service.VersionServiceApplyParams{
 		Apply:           "latest",
-		OperatorVersion: "1.15.0",
+		OperatorVersion: "1.16.0",
```

Comment:
in the line below `for _, v := range []string{"6.0", "5.0", "4.4"} {` please add `7.0` and remove `4.4`